listenbrainz/webserver/views/user.py

```diff
@@ -194,6 +196,54 @@ def reports(user_name: str):
         user=user
     )
 
+@user_bp.route("/<user_name>/playlists")
+def playlists(user_name: str):
+    """ Show user playlists """
+    
+    if not current_app.config.get("FEATURE_PLAYLIST", False):
+        raise NotFound()
+    
+    user = _get_user(user_name)
+    user_data = {
+        "name": user.musicbrainz_id,
+        "id": user.id,
+    }
+    
+    spotify_data = {}
+    current_user_data = {}
+    if current_user.is_authenticated:
+        spotify_data = spotify.get_user_dict(current_user.id)
+        current_user_data = {
+            "id": current_user.id,
+            "name": current_user.musicbrainz_id,
+            "auth_token": current_user.auth_token,
+        }
+    
+    loadPrivatePlaylists = False
+    if current_user.is_authenticated and \
+       current_user.musicbrainz_id == user_name:
```

Comment:
This isn't gonna work, looks like I'm comparing ID and name. I'll have a look.

Reply:
I think its correct. See Line 218